package.json

```diff
@@ -5,9 +5,11 @@
   "main": "index.js",
   "scripts": {
     "dev": "concurrently \"npm run client\" \"npm run server\"",
-    "build": "cd client && npm run build",
+    "build-client": "cd client && npm run build",
+    "build": "tsc",
     "client": "cd client && npm start",
-    "server": "cross-env NODE_ENV=development nodemon index",
+    "server": "cross-env NODE_ENV=development nodemon --watch \"./**\" --ext \"ts,json\" --exec \"ts-node index.ts\"",
+    "serve": "node ./dist/index",
```

Comment:
@alirezariahi Does it change anything in the build/dev process which is mention in the [doc](https://github.com/muke1908/chat-e2ee#installation)?

Reply:
@muke1908 No,